panels/site_sections/attractions_admin.py

```diff
@@ -324,6 +314,42 @@ def delete_event(self, session, id):
         if message:
             return {'error': message}
 
+    @ajax
+    def cancel_signup(self, session, id):
+        message = ''
+        if cherrypy.request.method == 'POST':
+            signup = session.query(AttractionSignup).get(id)
+            attraction_id = signup.event.feature.attraction_id
+            attraction = session.query(Attraction).get(attraction_id)
+            if not session.admin_attendee().can_admin_attraction(attraction):
+                message = "You cannot cancel a signup for an attraction you don't own"
+            elif signup.checkin_time:
+                message = "You cannot cancel a signup that has already checked in"
+            else:
+                session.delete(signup)
+                session.commit()
+        if message:
+            return {'error': message}
+
+    @only_renderable(c.STUFF, c.PEOPLE, c.REG_AT_CON)
```

Comment:
Maybe we should just rename the REG_AT_CON permission to "volunteer" 😅  That's out of scope for this PR, of course.